ZenPacks/zenoss/Microsoft/Windows/device_classes.yaml

```diff
@@ -90,6 +90,7 @@ device_classes:
       zDeviceTemplates: [Device, Cluster, Active Directory, IIS, MSExchange2010IS]
```

Comment:
There are a few files that changed their file mode. Was that planned change, or it happened accidentally?

Reply:
yeah, this happens due to where i'm editing my files.  sometimes i edit on my windows, sometimes in linux.  it doesn't affect the finished product